### DISCUSSION TITLE
Preview release option should keep working after preview window stops

### DISCUSSION THREAD

**Discussion by sjurgis** [forcedotcom/cli#2366] [Ideas]
Right now on every release start and end we have to update our CI process. That's 6 updates per year (12 because we have two trunks).
It would be much easier if either `"release": "preview",` wouldn't throw error after preview window is over. Right now it throws:
`ERROR running force:org:create:  At this time, we are outside of the preview period. You can create only current release scratch orgs.`

I'd much rather disable preview for rare occasion we need to, than keep opting in multiple times a year.

**Comment by yippie** [comment]
Yes please. We ended up building our own script to try preview, then if it fails, try regular.  It makes scratch org creation take longer but ensures that our developers and CI are testing on the latest version all the time.

**Comment by ImJohnMDaniel** [comment]
+1
